### DISCUSSION TITLE
login and password requested / where can i find them?

### DISCUSSION THREAD

**Discussion by michaelkaspy** [FlowiseAI/Flowise#202] [Q&A] [answered]
hello, would be cool if anyone can help. I am not developper and tryed to install flowise by installing node and then typing the command "npx flowise start". the flowise page is opening on localhost:3000 but there is a login popup coming up asking for username and password...what should i put there?

**Comment by LogicaldataCo** [comment]
In README File there been demo User: and Password  Check it

🔒 Authentication

To enable app level authentication, add USERNAME and PASSWORD to the .env file
in packages/server:

USERNAME=user
PASSWORD=1234

 I hope you got there

**Reply by michaelkaspy** [reply]
Thanks a lot but i already tryed those ones and they dont work for me ... i have looked everywhere to understand but i have no clue why...do u have any solution?

**Reply by michaelkaspy** [reply]
or maybe its just because i dont know how to do it correty...thats more than possible!...how can i open env file to modify the login/password on my local computer?

**Reply by caleboleary** [reply]
I had the same issue. In terminal I saw that USERNAME was already a globally defined env var for me and I think it was reading that, on windows I checked with ECHO %USERNAME% in cmd. I ended up getting in with what returned from that and then the default password "1234".

**Reply by PopAI22** [reply]
for me for user was the Github email and the Github password - it worked

**Comment by LogicaldataCo** [comment]
Docker Compose

   1. Go to docker folder at the root of the project
   2. Create .env file and specify the PORT (refer to .env.example)
   3. docker-compose up -d
   4. Open http://localhost:3000
   5. You can bring the containers down by docker-compose stop

<https://github.com/FlowiseAI/Flowise#docker-image>Docker Image

   1. Build the image locally:

   docker build --no-cache -t flowise .

   2. Run image:

   docker run -d --name flowise -p 3000:3000 flowise

   3. Stop image:

   docker stop flowise
   @Logicaldata Co ***@***.***> Did you used Docker ?

**Reply by michaelkaspy** [reply]
well i didnt use docker, i just installed flowise in node directly, and then i wrote the exe command to start it. so i dont know where its installed on my computer , when i  search for flowise it seems i dont find anything.
Do i need to use docker now even if i didnt install it with docker previously ? if yes, i dont need to uninstall smth?

**Comment by HenryHengZJ** [comment]
sorry for the bug! It is now fixed. Please reinstall or pull from latest (v1.2.10)

For npm: [README](https://github.com/FlowiseAI/Flowise#quick-start)

For docker: [README](https://github.com/FlowiseAI/Flowise/blob/main/docker/README.md)

**Reply by HenryHengZJ** [reply]
A little context of what happened: this is because by default, Windows has a variable `process.env.USERNAME`. This override the `USERNAME ` we set in `.env` file or via cli. Hence changing to a more specific name like `FLOWISE_USERNAME`

**Reply by michaelkaspy** [reply, marked as answer]
thanks a lot for your reactivity ! it works now! :)
